DirectProgramming/C++SYCL_FPGA/Tutorials/Features/experimental/hostpipes/README.md

```diff
@@ -407,37 +374,18 @@ using D2HPipe = cl::sycl::ext::intel::prototype::pipe<
 
 ### On Linux
 
-1. Run the sample on the FPGA emulator (the kernel executes on the CPU).
-   ```
-   ./hostpipes.fpga_emu
-   ```
-2. Run the sample on the FPGA simulator.
-   ```
-   CL_CONTEXT_MPSIM_DEVICE_INTELFPGA=1 ./hostpipes.fpga_sim <input_file> [-o=<output_file>]
-   ```
-3. Run the sample on the FPGA device (only if you ran `cmake` with `-DFPGA_DEVICE=<board-support-package>:<board-variant>`).
-   ```
-   ./hostpipes.fpga
-   ```
-
-### On Windows
-
-1. Run the sample on the FPGA emulator (the kernel executes on the CPU).
-   ```
-   hostpipes.fpga_emu.exe
-   ```
-2. Run the sample on the FPGA simulator.
-   ```
-   set CL_CONTEXT_MPSIM_DEVICE_INTELFPGA=1
-   hostpipes.fpga_sim.exe <input_file> [-o=<output_file>]
-   set CL_CONTEXT_MPSIM_DEVICE_INTELFPGA=
-   ```
-3. Run the sample on the FPGA device (only if you ran `cmake` with `-DFPGA_DEVICE=<board-support-package>:<board-variant>`).
-   ```
-   hostpipes.fpga.exe
-   ```
-
-## Example Output
+  * On Linux
```

Comment:
Please keep the original formatting for this section (having ### On linux and ### On windows)
This is missing emulator instructions and I don't think the program takes input/output files?